doc/source/reference/c-api.ufunc.rst

```diff
@@ -93,12 +93,18 @@ Functions
         the corresponding 1-d loop function in the func array.
 
     :param types:
-        Must be of length (*nin* + *nout*) \* *ntypes*, and it
-        contains the data-types (built-in only) that the corresponding
-        function in the *func* array can deal with.
+        Int8 of length `(nin + nout) * ntypes` It encodes the :ref:`dtype.num`
+        (built-in only) that the corresponding function in the `func` array
+        accepts. For a ufunc with two `ntypes`, one `nin` and one `nout` where
+        the first function accepts and returns `int32` and the second accepts
+        and returns `int64`, `types` would be `\05\05\07\07` since `int32.num`
```

Comment:
Double backticks for code snippets.

Would be clearer here to say ````types would be ``(char[]) {5, 5, 7, 7}`` ```, I think - no need to reference a string literal